org.mfat.systemdpilot/org.mfat.systemdpilot.yml

```diff
@@ -0,0 +1,37 @@
+app-id: org.mfat.systemdpilot
```

Comment:
https://mfat.org/ says 404 not found. How do I verify that you control it? Please put something up related to the app.

Reply:
I changed to io.github.mfat.systemdpilot.